csrc/fusion.h

```diff
@@ -146,6 +146,9 @@ class NVF_API Fusion : public IrContainer {
   //! Register input as an input of the fusion
   void addInput(Val* input);
 
+  //! Add output to outputs_ without modifying hide_output
+  void addOutputHelper(Val* output);
```

Comment:
```suggestion
  void addOutputInternal(Val* output);
```
just so readers know it's an internal method that should be used with care and shouldn't be exposed to the Python API.

Reply:
Will do. Since I'll also need to update a few other places. I'll manually change this.